CONTRIBUTING.md

```diff
@@ -11,7 +11,34 @@ To get the development and test environment set up on your local machine, you ne
 
 You are now ready to build and test Blazorise:
 
-## Running documentation
+## How to Write a Community Blog?
+
+To write a blog post, you must create a new subfolder under the `Pages/Blog` in the `Blazorise.Docs` project. The subfolder must contain a blog date and name in the format `YYYY-MM-DD_BlogName`, e.g., `2022-06-08_BeginnersGuideToCreateBlazoriseApp`. Note that the blog name must be unique.
+
+Next create the `Index.md` file under the same folder. The following metadata should always be present at the beginning of the file. Once pasted, adjust them for your blog.
+
+```
+---
+title: How to create a Blazorise WASM application: A Beginner's Guide
+description: Learn How to create a Blazorise WASM application: A Beginner's Guide.
+permalink: /blog/how-to-create-a-blazorise-application-beginners-guide
+canonical: /blog/how-to-create-a-blazorise-application-beginners-guide
+image-url: img/blog/2022-06-08/How_to_create_a_Blazorise_application_A_Beginners_Guide.png
+image-title: Blazorise WASM application: A Beginner's Guide
+author-name: Mladen Macanović
+author-image: mladen
+posted-on: June 8th, 2022
+read-time: 5 min
+---
+```
+
+Most of the settings should be self-explanatory. The only one that might need some extra work is the `author-image`. The image should be placed in the `wwwroot/img/avatars` under the `Blazorise.Docs.Server` project and in the `*.png` file format.
```

Comment:
I remember you compressed my image? Was it an extension? Worth mentioning?

Reply:
Good idea. Not an extension but an online tool. https://imagecompressor.com/